diff -r ea21b95926c7 go1/sqlite3/shell.c
--- a/go1/sqlite3/shell.c	Sat May 25 09:02:31 2013 -0400
+++ b/go1/sqlite3/shell.c	Sat May 25 09:03:57 2013 -0400
@@ -1,3 +1,5 @@
+// +build windows go1.1
+
 /*
 ** 2001 September 15
 **
@@ -2963,7 +2965,7 @@
   return argv[i];
 }
 
-int main(int argc, char **argv){
+int shell_main(int argc, char **argv){
   char *zErrMsg = 0;
   struct callback_data data;
   const char *zInitFile = 0;
diff -r ea21b95926c7 go1/sqlite3/sqlite3.c
--- a/go1/sqlite3/sqlite3.c	Sat May 25 09:02:31 2013 -0400
+++ b/go1/sqlite3/sqlite3.c	Sat May 25 09:03:57 2013 -0400
@@ -1,3 +1,5 @@
+// +build windows go1.1
+
 /******************************************************************************
 ** This file is an amalgamation of many separate C source files from SQLite
 ** version 3.7.17.  By combining all the individual C code files into this 
@@ -23442,6 +23444,9 @@
 /* Forward reference */
 static int openDirectory(const char*, int*);
 
+/* Fix for "error: 'fchmod' undeclared here (not in a function)" on FreeBSD 9 */
+int fchmod(int, mode_t);
+
 /*
 ** Many system calls are accessed through pointer-to-functions so that
 ** they may be overridden at runtime to facilitate fault injection during
