diff -r 3321ce4b52b7 go1/sqlite3/shell.c
--- a/go1/sqlite3/shell.c	Thu Apr 25 14:35:42 2013 -0400
+++ b/go1/sqlite3/shell.c	Thu Apr 25 14:36:07 2013 -0400
@@ -1,3 +1,5 @@
+// +build windows go1.1
+
 /*
 ** 2001 September 15
 **
@@ -2935,7 +2937,7 @@
   return argv[i];
 }
 
-int main(int argc, char **argv){
+int shell_main(int argc, char **argv){
   char *zErrMsg = 0;
   struct callback_data data;
   const char *zInitFile = 0;
diff -r 3321ce4b52b7 go1/sqlite3/sqlite3.c
--- a/go1/sqlite3/sqlite3.c	Thu Apr 25 14:35:42 2013 -0400
+++ b/go1/sqlite3/sqlite3.c	Thu Apr 25 14:36:07 2013 -0400
@@ -1,3 +1,5 @@
+// +build windows go1.1
+
 /******************************************************************************
 ** This file is an amalgamation of many separate C source files from SQLite
 ** version 3.7.16.2.  By combining all the individual C code files into this 
@@ -23260,6 +23262,9 @@
 /* Forward reference */
 static int openDirectory(const char*, int*);
 
+/* Fix for "error: 'fchmod' undeclared here (not in a function)" on FreeBSD 9 */
+int fchmod(int, mode_t);
+
 /*
 ** Many system calls are accessed through pointer-to-functions so that
 ** they may be overridden at runtime to facilitate fault injection during
